doc/samples/readme-encode_linux.md

```diff
@@ -74,7 +74,7 @@ The following command-line switches are optional:
   | [-num_active_BL0 numRefs] | number of maximum allowed references for B frames in L0 (for HEVC only)|
  |  [-num_active_BL1 numRefs] | number of maximum allowed references for B frames in L1 (for HEVC only)|
   | [-la] | use the look ahead bitrate control algorithm (LA BRC) (by default constant bitrate control method is used) <br> for H.264, H.265 encoder. Supported only with -hw option on 4th Generation Intel Core processors <br>if [-icq] option is also enabled simultaneously, then LA_ICQ bitrate control algotithm will be used.
- |[-lad depth] | depth parameter for the LA BRC, the number of frames to be analyzed before encoding. In range [10,100]. <br>may be 1 in the case when -mss option is specified <br>if [-icq] option is also enabled simultaneously, then LA_ICQ bitrate control algorithm will be used.|
+ |[-lad depth] | depth parameter for the LA BRC, the number of frames to be analyzed before encoding. May be in range[max(2*`GopRefDist`,2*`NumRefFrame`),100]. <br>may be 1 in the case when -mss option is specified <br>if [-icq] option is also enabled simultaneously, then LA_ICQ bitrate control algorithm will be used.|
```

Comment:
> max(2*`GopRefDist`,2*`NumRefFrame`)

emmm... This contradicts in-source help message and error checks which you have below. Can you, please, align document and in-source help (the latter does not mention -mss for example)?